tests/acceptance/test_full_long_covid_study.py

```diff
@@ -34,47 +43,47 @@ class Cohort:
 
     # Population
     # Patients registered on 2020-11-01
-    _registrations = table("practice_registrations").date_in_range(index_date)
+    _registrations = Table(WIP_PracticeRegistrations).date_in_range(index_date)
```

Comment:
It would be a kindness to provide something analogous to `tests.lib.frames` for all these `Table()` calls. But maybe you have avoided doing that because all the invocations will be removed or rewritten during the great test upheaval following Dave's work on the query model?